packages/components/src/InputFile/InputFile.tsx

```diff
@@ -115,23 +136,40 @@ export function InputFile({
   if (allowedTypes === "images") options.accept = "image/*";
   const { getRootProps, getInputProps, isDragActive } = useDropzone(options);
 
-  const { buttonLabel, hintText } = getLabels(allowMultiple, allowedTypes);
-  const dropZone = classnames(styles.dropZone, {
+  const { buttonLabel, hintText } = getLabels(
+    providedButtonLabel,
+    allowMultiple,
+    allowedTypes,
+  );
+  const dropZone = classnames(styles.dropZoneBase, {
```

Comment:
I wish we could add something to eslint to force a blank line after declaration.